.env.template

```diff
@@ -4,10 +4,13 @@ DATABASE_USERNAME=
 DATABASE_PASSWORD=
 DATABASE_PORT=5432
 
-# api
+# globals
 FRONTEND_BASE_URL=http://localhost:8080
+BACKEND_BASE_URL=http://localhost:3000
+
+FT_OAUTH2_CLIENT_ID=
```

Comment:
i guess you wanted to write 'SECRET' instead of 'ID'

Reply:
Agreed, one of them should probably be SECRET